### ISSUE TITLE
Comment is not added to rule

### ISSUE THREAD

**Issue by grrr2** [jeverley/dscpclassify#15]
hi,

I think there might be a bug regarding comment handling.

E.g. in create_user_rule() this is how you add comment
post_include `insert rule inet dscpclassify static_classify $nfproto $l4proto $oifname $dport $iifname $sport ${counter:+counter} $verdict ${comment:+comment \"$comment\"}` but actually it seems fetching comment option is missing from the code.
```
create_user_rule() {
        local enabled family proto direction device dest dest_ip dest_port src src_ip src_port counter class name
        local nfproto l4proto oifname daddr daddr6 dport iifname saddr saddr6 sport verdict comment

        config_get_bool enabled "$1" enabled 1
        [ "$enabled" = 1 ] || return 0
[..]
        config_get name "$1" name
        config_get comment "$1" comment  # <-- adding such way i think would solve this
```

**Comment by jeverley** [comment]
Well spotted @grrr2.

 I'd chosen the 'name' config option for rules so that the format aligned with https://openwrt.org/docs/guide-user/firewall/firewall_configuration , looks like I updated the rule insert lines to read a comment variable like the user ipsets at some point.

This should now be fixed with b9d6cb3, possible to retest?
You'll want to add the name as I've shown in the example config.

**Comment by grrr2** [comment]
thank you! it is working now.
